gadgets/trace_dns/program.bpf.c

```diff
@@ -190,6 +190,11 @@ struct {
 	__uint(max_entries, 1024);
 } query_map SEC(".maps");
 
+static __always_inline int min(int a, int b)
```

Comment:
`bpf_core_field_size` seems to return an unsigned int:

```
#define bpf_core_field_size(field)                                          \
        __builtin_preserve_field_info(field, BPF_FIELD_BYTE_SIZE)
```

https://github.com/llvm/llvm-project/blob/40cc4379cda6e0d6efe72c55d1968f9cf427a16a/clang/include/clang/Basic/BuiltinsBPF.td#L15

Should the `min()` signature be changed to unsigned `(int a, unsigned int b)`?